.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+ports:
+- port: 8000
+  onOpen: open-preview
+tasks:
+- init: npm install && npm install -g grunt-cli
+  command: grunt build && python3 -m http.server 8000
```

Comment:
if you're ok with doing hard refreshes, `grunt fastBuild` is a faster version. it skips the hashing step and simply puts the assets in build.js or something, which means you need to hard refresh but saves you a decent chunk of time

Reply:
Good idea! I'll follow up to use `grunt fastBuild` instead of `grunt build`. Saving time is 👌.